baseband/vdif/tests/test_vdif.py

```diff
@@ -648,33 +651,34 @@ def test_mwa_vdif():
     """Test phased VDIF format (uses EDV=0)"""
     with vdif.open(SAMPLE_MWA, 'rs', sample_rate=1.28*u.MHz) as fh:
         assert fh.samples_per_frame == 128
-        assert fh.frames_per_second == 10000
+        assert fh.sample_rate == 1.28*u.MHz
         assert fh.time == Time('2015-10-03T20:49:45.000')
         assert fh.header0.edv == 0
 
 
 def test_arochime_vdif():
     """Test ARO CHIME format (uses EDV=0)"""
+    sample_rate = 5**8*u.Hz   # 390625 frames/second * 1 samples/frame.
```

Comment:
If you want to understand where this comes from,
```
sample_rate = 800*u.MHz / 1024
```
(where one needs 1024=nchan*2 real samples for each FT)